.github/workflows/continuous-delivery.yml

```diff
@@ -61,7 +67,7 @@ jobs:
     name: Merge
     runs-on: ubuntu-latest
     needs: metadata
-    if: github.event.inputs.merge == 'y'
+    if: github.event.inputs.merge == true
```

Comment:
```suggestion
    if: github.event.inputs.merge
```
can't we do this?

Reply:
What i found online is that ==true is better. source: https://stackoverflow.com/questions/2953646/how-can-i-declare-and-use-boolean-variables-in-a-shell-script/21210966#21210966

Reply:
we need to decide what would be better, and implement that everywhere, i forgot it in one line i saw.

Reply:
You are talking about shell expressions, I'm talking about GitHub expressions: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions

They are separate concepts.

I've doublechecked why I suggested this. I come to a slightly more nuanced conclusion now, based on https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs
I think the conclusion is we can use `inputs.merge` and we don't need to check against a string, but when we use `github.event.inputs.merge` we do need to check against a string, because:
> The workflow will also receive the inputs in the github.event.inputs context. The information in the inputs context and github.event.inputs context is identical except that the inputs context preserves Boolean values as Booleans instead of converting them to strings. The choice type resolves to a string and is a single selectable option.

Reply:
I think implemented it correctly can you check?